docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -35,8 +37,23 @@
 
 \section{Team Meeting Plan}
 
+The team will have weekly team meetings, currently set to 10 am every Friday, though it can be 
+flexible and will be subject to change in the winter semester. These meetings will be held on 
+Discord. The purpose of these meetings are to realign the team to the current deliverables and to 
+assign tasks to each team member. An agenda of every team meeting will be written as a Github 
+issue, and tagged with the meeting number and meeting date. Each meeting will be archived and can 
+be easily accessed by searching for issues with the ``meeting notes'' tag. Additional meetings with 
+the supervisor will be held on a need to meet basis.
+
 \section{Team Communication Plan}
 
+The team will communicate primarily through Discord, Facebook Messenger, and Github Issues. Discord 
+and Facebook messenger will be used for quick communication between the team members, with the added 
```

Comment:
```suggestion
and Facebook Messenger will be used for quick communication between the team members, with the added 
```
Technically its official name is ["Messenger"](https://about.meta.com/technologies/messenger/). Could say Meta's Messenger if we want to super accurate.

Reply:
I'll just change it to Messenger. I forgot about that